README.md

```diff
@@ -1,20 +1,26 @@
-redux-actions
-=============
+# redux-actions
 
 [![build status](https://img.shields.io/travis/acdlite/redux-actions/master.svg?style=flat-square)](https://travis-ci.org/acdlite/redux-actions)
 [![npm version](https://img.shields.io/npm/v/redux-actions.svg?style=flat-square)](https://www.npmjs.com/package/redux-actions)
 
 [Flux Standard Action](https://github.com/acdlite/flux-standard-action) utilities for Redux.
 
-```js
+## Installation
+
+```bash
 npm install --save redux-actions
 ```
-```js
-import { createAction, handleAction, handleActions } from 'redux-actions';
-```
+
+If you don’t use npm, you may grab the latest [UMD](https://npmcdn.com/redux-actions@latest/dist) build from [npmcdn](https://npmcdn.com) (either a [development](https://npmcdn.com/redux-actions@latest/dist/redux-actions.js) or a [production](https://npmcdn.com/redux-actions@latest/dist/redux-actions.min.js) build). The UMD build exports a global called `window.ReduxActions` if you add it to your page via a `<script>` tag. We *don’t* recommend UMD builds for any serious application, as most of the libraries complementary to Redux are only available on [npm](http://npmjs.org).
```

Comment:
> Can we bold, CAPS, linkify, or monospace npm?

The second `npm` is linkified, I will do it for the first one as well, good catch.

> What are some libraries complementary to Redux that are only available on NPM? Would it be worth listing some?

Well, there are [tons of packages](https://www.npmjs.com/search?q=redux) 🙈 I've just adopted from [react-redux](https://github.com/reactjs/react-redux#installation).

> Also, when will the files on the CDN exist? When they are published?

They are available on npmcdn as soon as we are publishing this to npm. There is no additional effort for us.